code/game/objects/items/food/turkish.dm

```diff
@@ -0,0 +1,58 @@
+/obj/item/food/cig_kofte
+    name = "Çiğ köfte"
+    desc = "Bunun içinde gerçekten çiğ et mi var ?"
+    icon = 'icons/obj/food/turkish.dmi'
+    icon_state = "cig_kofte"
+    food_reagents = list(
+        /datum/reagent/consumable/nutriment = 2,
+		/datum/reagent/consumable/nutriment/protein = 4,
+		/datum/reagent/consumable/nutriment/vitamin = 5
+    )
+    tastes = list("tomato" = 1, "onion" = 1, "wheat" = 1, "pepper" = 1)
+    foodtypes = VEGETABLES | GRAIN
+    w_class = WEIGHT_CLASS_TINY
+    crafting_complexity = FOOD_COMPLEXITY_2
```

Comment:
```suggestion
/obj/item/food/cig_kofte
    name = "cig kofte"
    desc = "Bunun içinde gerçekten çiğ et mi var?"
    icon = 'icons/psychonaut/obj/food/turkish.dmi'
    icon_state = "cigkofte"
    food_reagents = list(
        /datum/reagent/consumable/nutriment = 2,
		    /datum/reagent/consumable/nutriment/protein = 4,
		    /datum/reagent/consumable/nutriment/vitamin = 5
    )
    tastes = list("tomato" = 1, "onion" = 1, "wheat" = 1, "pepper" = 1)
    foodtypes = VEGETABLES | GRAIN
    w_class = WEIGHT_CLASS_TINY
    crafting_complexity = FOOD_COMPLEXITY_2
```
descriptionlar olsa da isimlerde türkçe kullanma çiğ köfre biraz istisna olduğu için "cig kofte" olması daha iyi
açıklamayı beğenmedim daha iyi bir şey bulsan iyi olur
çiğ köftede protein yoktur heralde proteini alıp nutrimente versen daha mantıklı olur

Reply:
çiğköftede protein var olması lazım, internette öyle yazıyo tabi etli çiğköftemi etsiz mi öyle bilmiyom.